x-pack/test/functional/apps/security/doc_level_security_roles.js

```diff
@@ -75,9 +75,7 @@ export default function ({ getService, getPageObjects }) {
         expect(hitCount).to.be('1');
       });
       const rowData = await PageObjects.discover.getDocTableIndex(1);
-      expect(rowData).to.be(
-        'nameABC Companyname.keywordABC CompanyregionEASTregion.keywordEAST_iddoc1_indexdlstest_score0_type -'
-      );
+      expect(rowData).to.contain('EAST');
```

Comment:
So when there are changes in the `Document` row, this test no longer has to be adapted, since it's check is now more specific to an existing value